mappings/net/minecraft/world/gen/chunk/ChunkGeneratorType.mapping

```diff
@@ -23,7 +35,11 @@ CLASS net/minecraft/class_5284 net/minecraft/world/gen/chunk/ChunkGeneratorType
 	METHOD method_28005 getDefaultBlock ()Lnet/minecraft/class_2680;
 	METHOD method_28006 getDefaultFluid ()Lnet/minecraft/class_2680;
 	METHOD method_28007 getConfig ()Lnet/minecraft/class_5311;
+	METHOD method_28555 matchesPreset (Lnet/minecraft/class_5284$class_5307;)Z
+	METHOD method_28561 getSeaLevel ()I
+	METHOD method_28562 isMobGenDisabled ()Z
```

Comment:
Would `isMobSpawningDisabled` suffice here?

Reply:
It controls whether mobs will generate as part of the world I think, but I could mention that in a javadoc